Stripe/PublicHeaders/STPBackendAPIAdapter.h

```diff
@@ -37,7 +35,7 @@ typedef void (^STPCustomerCompletionBlock)(STPCustomer * __nullable customer, NS
  *  @see STPCard
  *  @param completion call this callback when you're done fetching and parsing the above information from your backend. For example, `completion(customer, nil)` (if your call succeeds) or `completion(nil, error)` if an error is returned.
  */
-- (void)retrieveCustomer:(STPCustomerCompletionBlock)completion;
+- (void)retrieveCustomer:(nullable STPCustomerCompletionBlock)completion;
```

Comment:
I wonder if there will be any unintended consequences of changing this API. If so, seems fairly straightforward for us to avoid it.